README.md

```diff
@@ -0,0 +1,4 @@
+# chuekbot
+This a discord bot build for the purposes of encouraging engagement and discussion in a discord group, aiding communication and exchanging valuable insights.
+
+To run this locally, please use the requirements.txt file, where project dependencies are specified.
```

Comment:
It is not clear what I need to run to install the project and run it locally. Here we should have bash commands.